scenario/post_processing/utils/visualization.py

```diff
@@ -0,0 +1,121 @@
+"""Visualization utils to create movie from VTK files."""
+import os
+import pathlib
+import tempfile
+from typing import List, Optional
+
+import pyvista as pv
+import moviepy.video.io.ImageSequenceClip as moviepy_io
+
+
+def get_sorted_files(data_dir: str,
```

Comment:
all this looks overly complicated? Why do we need this?

Reply:
Going to simplify, since the goal is simplicity.